app/_plugins/drops/plugins/schema.rb

```diff
@@ -76,19 +84,21 @@ def elements
               SchemaField.new(name: f.keys.first, parent: anchor, schema: f.values.first)
             end
             @schema['elements']
-          end
+          end.compact
         end
 
         def fields
           @fields ||= @schema.fetch('fields', []).map do |f|
             SchemaField.new(name: f.keys.first, parent: anchor, schema: f.values.first)
-          end
+          end.compact
         end
       end
 
       class Schema < Liquid::Drop
         extend Forwardable
 
+        DEPRECATION_MESSAGE = 'The following field has been deprecated and will be removed in a future version.'
```

Comment:
```suggestion
        DEPRECATION_MESSAGE = 'This field has been deprecated and will be removed in a future version.'
```
Since the banner is now in the description for the field, not before it.